components/ui/chart.tsx

```
@@ -169,75 +163,97 @@ const ChartTooltipContent = React.forwardRef<
         className={className}
       >
         {!nestLabel ? tooltipLabel : null}
+
         <div className="grid gap-1.5">
           {payload.map((item, index) => {
```

ESLint

[error] 'index' is defined but never used.

(@typescript-eslint/no-unused-vars)